### ISSUE TITLE
php8 attributes clones last attribute on methods

### ISSUE THREAD

**Issue by fernando-rivas-smtp** [prettier/plugin-php#1820]
**Prettier 2.1.2**

**PHP Plugin 0.17.4**

**Input:**

```php
<?php

declare(strict_types=1);

namespace App\Shared\Domain\Event;

final class DomainEventMessage
{
    #[Some(type: "some_type")]
    public function getPayload(): string
    {
        return "444";
    }

    public function getPayload1(): string
    {
        return "555";
    }

    public function getPayload2(): string
    {
        return "666";
    }
}

```

**Output:**

```php
<?php

declare(strict_types=1);

namespace App\Shared\Domain\Event;

final class DomainEventMessage
{
    #[Some(type: "some_type")]
    public function getPayload(): string
    {
        return "444";
    }

    #[Some(type: "some_type")]
    public function getPayload1(): string
    {
        return "555";
    }

    #[Some(type: "some_type")]
    public function getPayload2(): string
    {
        return "666";
    }
}
```

**Expected behavior:**
output just one method should have attributes.

**Comment by alexander-akait** [comment]
Can you run `npm ls @prettier/plugin-php`

**Comment by fernando-rivas-smtp** [comment]
```
@prettier/plugin-php@0.17.4 /Users/choz/PersonalSpace/plugin-php
└── (empty)
```

**Comment by alexander-akait** [comment]
Can you run this inside your project?

**Comment by alexander-akait** [comment]
No need, reproduced :+1:

**Comment by czosel** [comment]
Fixed in `v0.17.5` :tada:
